climada/util/value_representation.py

```diff
@@ -179,3 +177,66 @@ def value_to_monetary_unit(values, n_sig_dig=None, abbreviations=None):
         mon_val = [sig_dig(val, n_sig_dig=n_sig_dig) for val in mon_val]
 
     return (mon_val, name)
+
+
+def safe_divide(numerator, denominator, replace_with=np.nan):
+    """
+    Safely divide two arrays or scalars.
+
+    This function handles division by zero and NaN values in the numerator or denominator.
+    If the division results in infinity or NaN, it is replaced by a specified value.
+
+    Parameters
+    ----------
+    numerator : array-like or scalar
+        The numerator for division.
+    denominator : array-like or scalar
+        The denominator for division. Division by zero is handled safely.
```

Comment:
there are only tests for scalar and np.array, but it could be a just a list - right?

Reply:
yes, indeed. I added a test for lists too.